src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationServiceTest.java

```diff
@@ -155,6 +134,30 @@ public void shouldNotSendRoboticsEmailWhenCCJByDetermination() {
 
         service.notifyRobotics(event);
 
-        verifyNoInteractions(emailService);
+        verifyEmailSent();
+    }
+
+    private void verifyEmailSent() {
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
```

Comment:
I know this was just a refactor, but since we're here:
The captured `senderArgument` is never inspected - this could be replaced with either the precise expectation or `anyString()`. (Captors for Strings are very rarely needed.)

Reply:
Fixed